src/hooks/usePoolData.ts

```diff
@@ -127,7 +134,8 @@ export default function usePoolData(
         swapContract == null ||
         tokenPricesUSD == null ||
         library == null ||
-        chainId == null
+        chainId == null ||
+        migratorContract == null
```

Comment:
Let's double check that this contract exists on all chains (arbitrum) since this logic will break pools if not.

Reply:
Okay this is not / will not be deployed on Arbitrum, so we need to tweak this to allow it to be null while still loading the pool data and defaulting `isMigrated` to false. @ug02fast